tasks/gene_typing/variant_detection/task_ivar_variant_call.wdl

```diff
@@ -36,28 +32,27 @@ task variant_call {
     
     # set reference gff
     if [[ ! -z "~{reference_gff}" ]]; then
-      echo "User reference identified; ~{reference_genome} will be utilized for alignement"
+      echo "User reference identified; ~{reference_gff} will be utilized for alignement"
```

Comment:
utilized is grammatically incorrect; both of these should say "used" 

grammar lesson: utilized is when you're using something _not_ for its original purpose. used is when you are. 
i.e., i utilized a microwave to fire my pottery instead of a kiln vs. i used a microwave to heat up some food